support-models/src/test/scala/com/gu/support/workers/SendThankYouEmailStateSerialisationSpec.scala

```diff
@@ -38,7 +38,7 @@ object ProductTypeCreatedTestData {
 
   val digitalSubscriptionCreated = SendThankYouEmailDigitalSubscriptionState(
     user = User("111222", "email@blah.com", None, "bertha", "smith", Address(None, None, None, None, None, Country.UK)),
-    DigitalPack(GBP, Monthly, ReaderType.Direct),
```

Comment:
Can the `ReaderType` object go? I don't see it being deleted in this PR.

Reply:
No it is still needed for GW gifts for example and we may end up using it for students at some point